app/views/home/home.html.erb

```diff
@@ -0,0 +1,20 @@
+<section id="content">
+  <ol id="links-list">
+    <li>
+      <span class="link">
+        <%= link_to "The coolest thing you've ever seen.", "#" %>
+      </span>  
+      <span class="details">
+        <p> 
+          posted by
+          <%= link_to "mr.man", "#" %> 
+          5 minutes ago
+        </p>
+      </span>
+    </li>
+  </ol>
+</section> 
```

Comment:
If this is the main content of the page, checkout the [`<main />` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main).